arviz/data/inference_data.py

```diff
@@ -228,15 +237,18 @@ def sel(self, inplace=False, chain_prior=False, warmup=False, **kwargs):
 
         Parameters
         ----------
-        inplace : bool, optional
+        groups: str or list of str, optional
+            Groups where the selection is to be applied. Can either be group names
+            or metagroup names.
+        inplace: bool, optional
             If ``True``, modify the InferenceData object inplace,
             otherwise, return the modified copy.
-        chain_prior: bool, optional
+        chain_prior: bool, optional, deprecated
             If ``False``, do not select prior related groups using ``chain`` dim.
-            Otherwise, use selection on ``chain`` if present.
-        warmup: bool, optional
-            If ``False``, do not select warmup groups.
-        **kwargs : mapping
+            Otherwise, use selection on ``chain`` if present. Default=False
+        warmup: bool, optional, deprecated
```

Comment:
is this in the released version?

Reply:
I don't think so, it's true if we manage to merge before the release we can just remove it.